roles/satellite/tasks/internals/activationkey-hostgroup-definition.yml

```diff
@@ -37,6 +37,8 @@
     parameters:
     - name: "kt_activation_keys"
       value: "{{ g_test_activationkey }}"
+    - name: "keyboard"
+      value: "{{ keyboard | default(omit) }}"
```

Comment:
You sure this will work as intended?

The `value` field will be removed but not the `name` so you will have effectively:

```
-  name: "keyboard"
```

Reply:
Updated with the default value 'us'